runtime/vm/LayoutFFITypeHelpers.cpp

```diff
@@ -203,7 +203,12 @@ LayoutFFITypeHelpers::getStructFFITypeElements(char **layout)
 void
 LayoutFFITypeHelpers::freeStructFFIType(ffi_type *ffiType)
 {
-	if ((NULL != ffiType) && (FFI_TYPE_STRUCT == ffiType->type)) {
+	if ((NULL != ffiType) && ((FFI_TYPE_STRUCT == ffiType->type)
+#if defined(J9ZOS390)
+			|| (FFI_TYPE_STRUCT_FF == ffiType->type)
+			|| (FFI_TYPE_STRUCT_DD == ffiType->type)
+#endif /* defined(J9ZOS390) */
+	 )) {
```

Comment:
The condition should be split before `&&` and `)) {` should be indented the same as `if`:
```c
	if ((NULL != ffiType)
		&& ((FFI_TYPE_STRUCT == ffiType->type)
#if defined(J9ZOS390)
			|| (FFI_TYPE_STRUCT_FF == ffiType->type)
			|| (FFI_TYPE_STRUCT_DD == ffiType->type)
#endif /* defined(J9ZOS390) */
	)) {
```